proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -57,6 +57,8 @@ message ShardInfo {
     map<string, google.protobuf.Timestamp> cluster_timer_ack_level = 11 [(gogoproto.stdtime) = true];
     map<string, int64> cluster_replication_level = 12;
     map<string, int64> replication_dlq_ack_level = 13;
+    int64 visibility_ack_level = 14;
+    map<string, int64> cluster_visibility_level = 15;
```

Comment:
the new visibility queue (or not sure whether calling it external queue being better?)
is dedicated for local cluster only
meaning `map<string, int64> cluster_visibility_level = 15;` will not be used until task multi cursor is implemented

i am fine keeping this or removing this attr

Reply:
I would better remove it for now.